lightpath/launcher.py

```diff
@@ -4,16 +4,16 @@
 import coloredlogs
 
 import happi
-import pydm
-
 import lightpath
+import pydm
```

Comment:
This is now the wrong sort order. The original order was correct. I'm not very pleased with isort based on this and other instances...